test/unit/demo/models/pond.test.js

```diff
@@ -0,0 +1,51 @@
+const {initFishData} = require('../../../../src/utils/fishData');
```

Comment:
Can we use some kind of root path to avoid all of these `..`?

Reply:
➕ we should be able to add `alias`es to our webpack config to do this -- [example in code-dot-org](https://github.com/code-dot-org/code-dot-org/blob/988ae4a9e030d05eb917649ddcb6cadaf94406af/apps/webpack.js#L32-L92)

Reply:
Maddie did this in #332 and I've updated these tests (not any other tests though)